apm-agent-plugins/apm-hibernate-search-plugin/apm-hibernate-search-plugin-6_x/src/main/java/co/elastic/apm/agent/hibernate/search/v6_x/HibernateSearch6Instrumentation.java

```diff
@@ -79,7 +79,7 @@
 
     @Override
     public Collection<String> getInstrumentationGroupNames() {
-        return Arrays.asList(HibernateSearchConstants.HIBERNATE_SEARCH_ORM_TYPE, "incubating");
+        return Arrays.asList(HibernateSearchConstants.HIBERNATE_SEARCH_ORM_TYPE, "experimental");
```

Comment:
I'd add both `incubating` and `experimental`. Otherwise, it's a breaking change for people that have set `disable_instrumentation=incubating,foo`.

Reply:
Good point! 
Not really related to this location- I will internally treat `incubating` as if it is an `experimental` tag